benchmarks/runbenchmarks.jl

```diff
@@ -0,0 +1,28 @@
+using BenchmarkTools
+using QuantumToolbox
+
+suite = BenchmarkGroup()
+
+suite["steadystate"] = BenchmarkGroup(["steadystate"])
+
+
+## steadystate ##
```

Comment:
There might be many benchmarks in the future. Should we separate each benchmark topic into individual files and `include` them in this file? 

To avoid messing the in-scope variables, maybe can put each benchmark into a function and just return `@benchmarkable XXX()`
and call those functions in this file:
```julia
suite["XXX"]["YYY"] = f(...)
```

What do you think ?

Reply:
Yes I agree, this is just to have everything minimal for the moment, and see if everything work. Then we should add as many benchmarks as we want, each one in a separated file.

Reply:
Okay~